CHANGELOG.md

```diff
@@ -13,6 +13,12 @@ _Note: Gaps between patch versions are faulty, broken or test releases._
 
 See [CHANGELOG - 6to5](CHANGELOG-6to5.md) for the pre-4.0.0 version changelog.
 
+## Unreleased
+
+  * **Internal**
+
+    * `babel-types`: Start test suite.
+
```

Comment:
Hmm yeah, it would be an interesting idea to add to just add to the changelog in the PR itself